wordpress-mobile/WordPress-iOS
# Comment: store raw content

Ref: #16906, #16876

This stores the `rawContent` property in the `Comment` model, and uses it when displaying:
- Comment snippet in Comments list.
- Comment content in Comment details.
- Comment content when editing a Comment.

`rawContent` contains markup for Gutenberg comments, so that is stripped when the Comment snippets are displayed via My Site > Comments, but is still displayed when editing Comments.

To test:
There should be no visible change where Comment content is displayed. So verify Comments content is displayed as it was before in:

- My Site > Comments.
- My Site > Comments > Comment details.
- My Site > Comments > Comment details > edit.
- Reader > post comments.
- Notifications > Comments > notification details.

Please check P2 and non-P2 sites.

## Regression Notes
1. Potential unintended areas of impact
Comments could be displayed incorrectly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Compared Comment displays with previous version of the app to verify they are the same.

3. What automated tests I added (or what prevented me from doing so)
I'll address in a future PR.

PR submission checklist:
